src/plugins/timeline/pluginSpec.js

```diff
@@ -228,4 +228,122 @@ describe('the plugin', function () {
         });
     });
 
+    describe("The timestrip composition policy", () => {
+        it("allows composition for plots", () => {
+            const parent = {
```

Comment:
since you use this more than once, maybe pull it up in scope and change the name to `timestripTelemetryObject`?

Reply:
Done